enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/compute/ComputeInstanceNode.java

```diff
@@ -45,13 +56,69 @@ public ComputeInstanceNode(ComputeInstanceItem instance) {
         setName(instance.getName());
         setDisplayName(instance.getName());
         setIconBaseWithExtension(COMPUTE_INSTANCE_ICON);
-        setShortDescription(Bundle.CoputeInstanceDesc(instance.getName()));
+        setShortDescription(Bundle.CoputeInstanceDesc(
+                instance.getName(), 
+                instance.getPublicIp(),
+                instance.getUsername(),
+                instance.getProcessorDescription()
+        ));
     }
 
     public static NodeProvider<ComputeInstanceItem> createNode() {
-        return ComputeInstanceNode::new;
+        return (instance) -> {
+            update(instance);
+            return new ComputeInstanceNode(instance);
+        };
     }
+    
+    
+    static public void update(ComputeInstanceItem instance) {
```

Comment:
Not checked alive, but make sure this runs in other thread than EDT that usually expands nodes and a different thread than LSP event one.

Reply:
The `ChildFactory` class is used to create children nodes asynchronously. This ensures that the `update()` method is called outside the EDT thread, thereby preventing any blocking operations on the EDT.